dev/release/verify-release-candidate.sh

```diff
@@ -622,7 +637,7 @@ test_linux_wheels() {
     for tag in ${platform_tags}; do
       # check the mandatory and optional imports
       pip install --force-reinstall python-rc/${VERSION}-rc${RC_NUMBER}/pyarrow-${VERSION}-cp${py_arch//[mu.]/}-cp${py_arch//./}-${tag}.whl
-      INSTALL_PYARROW=OFF ${ARROW_DIR}/ci/scripts/python_wheel_unix_test.sh ${ARROW_DIR}
+      INSTALL_PYARROW=OFF ${ARROW_DIR}/ci/scripts/python_wheel_unix_test.sh ${ARROW_TMPDIR}/apache-arrow-${VERSION}
```

Comment:
How about removing the existing `ARROW_DIR="..."` and defining `ARROW_DIR` in `get_or_decompress_source()` instead`?

Reply:
Done.

Reply:
Can we revert this change?

Reply:
Done